### DISCUSSION TITLE
Apply `$navigationLabel` for heading and add new button

### DISCUSSION THREAD

**Discussion by datlechin** [filamentphp/filament#3462] [Ideas]
![Screenshot 2022-08-10 at 21 54 06](https://user-images.githubusercontent.com/56961917/183935783-c16df354-1521-4646-ae07-e1e877addb6c.png)

My custom `$nvaigationLabel`:

```php
protected static ?string $navigationLabel = 'Người dùng';
```

The heading should be **Nguời dùng**, and the button should be **Tạo mới Người dùng**

**Comment by ryangjchandler** [comment]
The `navigationLabel` is separate from the `label` of the page.

**Reply by datlechin** [reply]
> The `navigationLabel` is separate from the `label` of the page.

ah, i see, thank you
